src/OrbitQt/CallTreeWidget.cpp

```diff
@@ -718,7 +737,7 @@ void CallTreeWidget::OnCustomContextMenuRequested(const QPoint& point) {
                         const orbit_client_data::CallstackEvent& callstack_event) -> bool {
                       return callstack_event.thread_id() != first_callstack_event.thread_id();
                     });
-    app_->SelectCallstackEvents(
+    app_->InspectCallstackEvents(
```

Comment:
Eventually we'll need to rename `kActionSelectCallstacks` and change its string to "Inspect". Add a TODO linking the feature bug?

Reply:
Instead, I just added the change in this PR based on whether the time_range_selection flag was enabled and moved the logic of Inspect vs Select to CallTreeWidget rather than OrbitApp. Hopefully that makes sense.